operator/config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: fmuyassarov/ansible
-  newTag: v10
```

Comment:
IMO these should be squashed into the original commit that introduces the operator.